src/components/elements/About/About.jsx

```diff
@@ -0,0 +1,122 @@
+import React from "react";
+import Title from "../../UI/Title/Title.jsx";
+
+import styles from "./About.module.scss";
+
+import icon_1 from "../../../assets/images/icon-about-1.svg";
+import icon_2 from "../../../assets/images/icon-about-2.svg";
+import icon_3 from "../../../assets/images/icon-about-3.svg";
+import icon_4 from "../../../assets/images/icon-about-4.svg";
+
+import image_1 from "../../../assets/images/image-about-1.jpg";
+import image_2 from "../../../assets/images/image-about-2.jpg";
+import image_3 from "../../../assets/images/image-about-3.jpg";
+import image_4 from "../../../assets/images/image-about-4.jpg";
+
+const About = () => {
+  return (
+    <section className={styles.about}>
+      <Title classes="title_brown">About Us</Title>
+
+      <div className={styles.about__container}>
```

Comment:
Combine all divs into one with one className with the combined styles

Reply:
I need clarification and one code example.

Reply:
Something like that 
```
 <div className={`${styles.card} ${props.className}`}>{props.children}</div>;
 ```